tfx/components/example_gen/write_split.py

```diff
@@ -13,22 +13,26 @@
 # limitations under the License.
 """PTransform for write split."""
 import os
-from typing import Union
+from typing import Union, Dict, Any
 
 import apache_beam as beam
+import pyarrow as pa
 import tensorflow as tf
 from tfx.proto import example_gen_pb2
 from tfx_bsl.telemetry import util
 
 # Default file name for TFRecord output file prefix.
-DEFAULT_FILE_NAME = 'data_tfrecord'
+from tfx.types import standard_component_specs
+
+DEFAULT_PARQUET_FILE_NAME = 'data_parquet'
+DEFAULT_TF_RECORD_FILE_NAME = 'data_tfrecord'
 
 # Metrics namespace for ExampleGen.
 METRICS_NAMESPACE = util.MakeTfxNamespace(['ExampleGen'])
 
 
 @beam.typehints.with_input_types(
-    Union[tf.train.Example, tf.train.SequenceExample, bytes])
+    Union[tf.train.Example, tf.train.SequenceExample, bytes, pa.Table])
```

Comment:
does this have to be updated to serialize dicts?

Reply:
we actually do not use the MaybeSerialize if we are using parquet, so gonna remove this change